packer/install-dependencies.sh

```diff
@@ -363,6 +363,19 @@ for KUBERNETES_VERSION in ${K8S_VERSIONS}; do
     echo "  - ${CONTAINER_IMAGE}" >> ${RELEASE_NOTES_FILEPATH}
 done
 
+# TODO: fetch supported k8s versions from an aks-engine command instead of hardcoding them here
+AZURE_STACK_K8S_VERSIONS="
+1.11.7
```

Comment:
we usually support the latest two patches of each minor version, is supporting 1.11.7 a requirement for you or would 1.11.8 and 1.11.9 enough?

Reply:
Our plan is to support two patches per minor version as well. 
It was unclear to me if 1.11.7 was still needed. Checked with Hongbin.